docs/v1/P4Runtime-Spec.mdk

```diff
@@ -1289,12 +1292,32 @@ message PkgInfo {
   string url = 8;
   // Miscellaneous metadata, structured; a way to extend PkgInfo
   repeated StructuredAnnotation structured_annotations = 9;
+  // If set, specifies the properties that the underlying platform should have.
+  // If the platform does conform to these properties, the server should reject
```

Comment:
does -> doesn't?

Reply:
Nice catch! Didn't copy from the proto when I fixed it :'(.